webpack.config.js

```diff
@@ -265,12 +271,6 @@ var config = {
             filename: 'root.html',
             inject: 'head',
             template: 'root.html',
-            meta: {
```

Comment:
This isn't actually needed since the server sets the CSP header as long as you aren't using `npm run dev-server`, and if you are, it's development, so security doesn't really matter